### ISSUE TITLE
Custom Sensors?

### ISSUE THREAD

**Issue by stuu3k** [netdata/netdata-grafana-datasource-plugin#26]
Hello,

I am having problems monitoring WAN \ LAN on my OPNSense routers with these, please see the below charts...
![image](https://github.com/netdata/netdata-grafana-datasource-plugin/assets/98365629/1a232b2a-e87c-4ef3-8688-dabf8d5a2eba)

Top is using the NET.NET Sensor grabbed by the plugin, Blue Input Orange Output, however instead of showing me figures for each it adds them showing a todal NET which is no good, also it is not interface specific which is a problem.

The Second chart reflects actual input output with Input in Green and Output in Yellow, problem is this is using a custom grabber and i can't control the data properly.

Can we please have a way to tell it which chats we want to pull from NetData, the old grabber I was using with other limitations lets me specify a list of charts like below, if only I could tell this plugin what metrics to grab this would be perfect.

     system.net mem.available mem.committed net.igb3 net.igb0 net.wg0

Looks amazing btw, this is  a Multi CPU Monitor I made in Grafana and the level of detail i get it amazing, if I can get this for NICs on a server this would be a game changer. :)
https://snapshots.raintank.io/dashboard/snapshot/pE81DTrdbeGq6WNecKadAQIkM1Nfb3T2

Thanks

**Comment by stuu3k** [comment]
Just to clarify and add this as I left it out, but for example with Custom Sensors I have some servers with eth0, some with vtnet0 some with opt0 etc. I want to be able to set a variable to add this to the collection metrics please.

Thanks

**Comment by hugovalente-pm** [comment]
thanks for the feedback, I believe what we need to add here now is the ability that was recently added on Netdata Cloud with the NIDL framework

filter by instance (the charts as you mention) and in `net.net` it is interfaces
![image](https://github.com/netdata/netdata-grafana-datasource-plugin/assets/82235632/a2f76789-c444-442e-aa85-ebeda020b605)

and labels
![image](https://github.com/netdata/netdata-grafana-datasource-plugin/assets/82235632/2f5c14c8-49c0-447c-ba16-bb561a59a133)

we'll consider this for a next cycle of things to bring to the plugin

**Comment by stuu3k** [comment]
Yes that sounds right, thanks for adding this to consideration and thanks again for your good work so far! :)

**Comment by novykh** [comment]
Should be added.
